pkgs/development/mobile/genymotion/default.nix

```diff
@@ -10,10 +10,10 @@ let
   libPath = "${stdenv.lib.makeLibraryPath packages}";
 in
 stdenv.mkDerivation rec {
-  name = "genymotion-${version}";
+  pname = "genymotion";
   version = "2.8.0";
   src = requireFile {
-    url = https://www.genymotion.com/download/;
+    url = "https://dl.genymotion.com/releases/genymotion-${version}/genymotion-${version}-linux_x64.bin";
```

Comment:
Actually, this URL points to the file directly, which we could just download with `fetchurl` anyways, no need for `requireFile`!

Reply:
The file is "unfree" so I thought it was necessary to have the user download it manually. If this is not the case I am happy to swap that out for `fetchurl`.

Reply:
Yeah unfree doesn't mean we can't download it like that, lots of packages in nixpkgs do that. If the file is available just like that this seems to indicate they don't want users to have to sign up to download it, so this should be fine. Unless they somewhere mention something special regarding that on their website (I haven't checked).

Reply:
Perfect I will do another squash.